src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -355,14 +357,13 @@ def _create_osd(self, svc_arg=None, inbuf=None):
                 return HandleCommandResult(-errno.EINVAL, stderr=msg)
 
             devs = DeviceSelection(paths=block_devices)
-            drive_group = DriveGroupSpec(node_name, data_devices=devs)
+            drive_groups = [DriveGroupSpec(node_name, data_devices=devs)]
         else:
             return HandleCommandResult(-errno.EINVAL, stderr=usage)
 
-        completion = self.create_osds(drive_group)
+        completion = self.create_osds(drive_groups)
```

Comment:
as you're changing the interface, please quickly adapt the orchestrator interface and all users of it (rook, ansible, test_orchestrator).